README.md

```diff
@@ -25,33 +24,81 @@ for your environment:
 
     cp conf/data.yml.example conf/data.yml
 
-There are a few environment variables that need to be set to run these scripts (see `conf/webindex-env.sh.example`
-for a list).  If you don't want to set them in your `~/.bashrc`, create `webindex-env.sh` in `conf/` and set them.
+There are a few environment variables that need to be set to run these scripts (see 
+`conf/webindex-env.sh.example` for a list).  If you don't want to set them in your `~/.bashrc`, 
+create `webindex-env.sh` in `conf/` and set them.
 
     cp conf/webindex-env.sh.example conf/webindex-env.sh
 
+### Download the paths file for a crawl
+
+For each crawl of the web, CommonCrawl produces a file containing a list of paths to the data 
+files produced by that crawl.  Webindex uses this file to retrieve CommonCrawl data stored 
+in S3. The command below downloads this paths file for the April 2015 crawl (identified by 
+`2015-18`) to the `paths/` directory as it will be necessary for future commands.  If you 
+would like to use a different crawl, the [CommonCrawl website][crawl] has a list of possible 
+crawls which are identified by the `YEAR-WEEK` (i.e. `2015-18`) of the time the crawl 
+occurred.
+
+    ./bin/webindex getpaths 2015-18
+
+Take a look at the paths file that was just retrieved:
+
+    $ less paths/2015-18.wat.paths 
+
+Each line in the paths file contains a path to a different common crawl data file.  In later 
+commands, you will select paths by specifying a range (in the format of `START-END`).  Ranges
+can start at index 0 and their start/end points are inclusive.  Thefore, a range `4-6` would
+selct 3 paths. Using the command below, you can find the max endpoint for ranges in a paths 
+file.
+
+    $ wc -l paths/2015-18.wat.paths 
+    38609 paths/2015-18.wat.paths
+
+The 2015-18 paths file has 38609 different paths.  A range of `0-38608` would select all 
+paths in the file.
+
 ### Copy CommonCrawl data from AWS into HDFS
 
-CommonCrawl data sets are hosted in S3.  The following command will run a Spark job that copies
-data files from AWS into HDFS `init/` and `load/` directoires.  This script is configured by 
-`conf/data.yml` where you can specify the number of files to copy into each directory:
+After you have retrieved a paths file, you can run the command below to run a Spark job that copies 
+data files from S3 to a directory in your local HDFS instance.  The command below will copy 3 files
+in the file range of `4-6` of the `2015-18` paths file into the HDFS directory `/cc/data/a`.  
+CommonCrawl data files are large (~330 MB each) so be careful not copy too many if you are running 
+in a development environment.
 
-    ./bin/webindex copy
+    ./bin/webindex copy 2015-18 4-6 /cc/data/a
 
-### Initialize Fluo & Accumulo with data
+If you need to create multiple data sets, you can run this command with different ranges to create 
```

Comment:
Could be

    To create multiple data sets, run the command with a different range and HDFS directory: